components/more_direct_channels/last_post_at.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {FunctionComponent} from 'react';
+
+import Timestamp from 'components/timestamp/timestamp';
+type Props = {
+    lastPostAt?: number;
+}
+
+export const LastPostAt: FunctionComponent<Props> = ({lastPostAt}: Props): JSX.Element | null => {
+    if (!lastPostAt) {
+        return null;
+    }
+
+    return (
+        <Timestamp
+            value={lastPostAt}
+            useTime={false}
+            style='long' //eslint-disable-line react/style-prop-object
+            ranges={[
```

Comment:
Can we extract the ranges into a constant as well?

Reply:
You mean the array as a local constant ?

Reply:
Yes